coding-the-legislation/bootstrap_a_new_country.md

```diff
@@ -1,11 +1,7 @@
 # Bootstrap a new country
```

Comment:
IMO there is a slight difference : 

A country package is a python package that provides a constructor named `CountryTaxBenefitSystem`, which can be used to instantiate a tax and benefit system.

But it also contains tests, as well as `reforms` which can be used to build other _slightly different_ tax and benefit systems. I'd say that the tax and benefit system is included in the country package.